charts/grafana/templates/image-renderer-deployment.yaml

```diff
@@ -45,11 +45,11 @@ spec:
       {{- end }}
       {{- if .Values.imageRenderer.securityContext }}
       securityContext:
-      {{ toYaml .Values.imageRenderer.securityContext | indent 2 }}
+        {{- toYaml .Values.imageRenderer.securityContext | nindent 8 }}
       {{- end }}
       {{- if .Values.imageRenderer.hostAliases }}
       hostAliases:
-      {{ toYaml .Values.imageRenderer.hostAliases | indent 2 }}
+        {{– toYaml .Values.imageRenderer.hostAliases | nindent 8 }}
```

Comment:
==> Linting charts/grafana
Error:  templates/: parse error at (grafana/templates/image-renderer-deployment.yaml:52): U+2013 '–' in command

Reply:
Fixed!

(I have no idea how the hyphen sign got replaced with U+2013 oO)